Tests/Reader/reader.cpp

```diff
@@ -0,0 +1,45 @@
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include <string>
+#include <sstream>
+#include <cmath>
+using namespace std;
+
+namespace COLUMNS {
+  constexpr int rad = 0, rho = 1, temp=2, ye = 3, eps = 4, vel = 5, press = 6, rho_adm = 7, P_adm =8, S_adm = 9, Srr_adm = 10;
+ }
```

Comment:
You might want to move this to `reader.hpp` so it's available everywhere you might want it.